### PR TITLE
KY Job Listing

### PR DESCRIPTION
**Pull request by kiyonggg**
- Added active states
- Added filtering function

### CODE REVIEW

File: src/components/KYSinglePriceGridComponent/SignUpButton.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { Button } from 'antd'
+
+export const SignUpButton = () => {
+  // add style={} in Button
```
Comment (xinyee20) [inline src/components/KYSinglePriceGridComponent/SignUpButton.tsx]: Comments can be removed

File: src/components/KYSinglePriceGridComponent/SignUpButton.tsx 01-47-11-159.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { Button } from 'antd'
+
+export const SignUpButton = () => {
+  // add style={} in Button
+  return <Button type="primary">Sign Up</Button>
+}
+
+export default SignUpButton
```
Comment (xinyee20) [inline src/components/KYSinglePriceGridComponent/SignUpButton.tsx 01-47-11-159.tsx]: Is this a repeated file?

**Review by xinyee20 — COMMENTED** [review]
When the tags are selected, perhaps its better to highlight all the selected tags in the applicable card rather than just the one being currently clicked?

File: src/components/KYSinglePriceGridComponent/SignUpButton.tsx 01-47-11-159.tsx (hunk 2 above)
Comment (kiyonggg) [reply src/components/KYSinglePriceGridComponent/SignUpButton.tsx 01-47-11-159.tsx]: deleted; resolved in commit df9457e

File: src/components/KYSinglePriceGridComponent/SignUpButton.tsx (hunk 1 above)
Comment (kiyonggg) [reply src/components/KYSinglePriceGridComponent/SignUpButton.tsx]: deleted; resolved in commit df9457e

**Review by xinyee20 — APPROVED** [review]
lgtm!

**Pull request merged**